.github/workflows/version-up.yml

```diff
@@ -0,0 +1,186 @@
+          -F files[][path]="pallets/orml-currencies-allowance-extension/Cargo.toml" -F files[][contents]=$(base64 -w0 pallets/orml-currencies-allowance-extension/Cargo.toml) \
+          -F files[][path]="pallets/orml-tokens-management-extension/Cargo.toml" -F files[][contents]=$(base64 -w0 pallets/orml-tokens-management-extension/Cargo.toml) \
+          -F files[][path]="pallets/parachain-staking/Cargo.toml" -F files[][contents]=$(base64 -w0 pallets/parachain-staking/Cargo.toml) \
+          -F files[][path]="pallets/parachain-staking/rpc/Cargo.toml" -F files[][contents]=$(base64 -w0 pallets/parachain-staking/rpc/Cargo.toml) \
+          -F files[][path]="pallets/parachain-staking/rpc/runtime-api/Cargo.toml" -F files[][contents]=$(base64 -w0 pallets/parachain-staking/rpc/runtime-api/Cargo.toml) \
+          -F files[][path]="pallets/treasury-buyout-extension/Cargo.toml" -F files[][contents]=$(base64 -w0 pallets/treasury-buyout-extension/Cargo.toml) \
+          -F files[][path]="pallets/vesting-manager/Cargo.toml" -F files[][contents]=$(base64 -w0 pallets/vesting-manager/Cargo.toml) \
+          -F files[][path]="runtime/amplitude/Cargo.toml" -F files[][contents]=$(base64 -w0 runtime/amplitude/Cargo.toml) \
+          -F files[][path]="runtime/common/Cargo.toml" -F files[][contents]=$(base64 -w0 runtime/common/Cargo.toml) \
+          -F files[][path]="runtime/foucoco/Cargo.toml" -F files[][contents]=$(base64 -w0 runtime/foucoco/Cargo.toml) \
+          -F files[][path]="runtime/integration-tests/Cargo.toml" -F files[][contents]=$(base64 -w0 runtime/integration-tests/Cargo.toml) \
+          -F files[][path]="runtime/pendulum/Cargo.toml" -F files[][contents]=$(base64 -w0 runtime/pendulum/Cargo.toml) \
+          -F 'query=@.github/api/createCommitOnBranch.gql'
+
+      - name: Commit New Changes to New Branch (node)
+        if: github.event.inputs.packages == 'node'
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BRANCH: ${{ steps.new-branch.outputs.name }}
+        run: |
+          gh api graphql \
+          -F githubRepository=${{ github.repository }} \
+          -F branchName=${{ env.BRANCH }} \
+          -F expectedHeadOid=$(git rev-parse HEAD) \
+          -F commitMessage="$(cat changes.txt)" \
+          -F files[][path]="node/Cargo.toml" -F files[][contents]=$(base64 -w0 node/Cargo.toml) \
+          -F 'query=@.github/api/createCommitOnBranch.gql'
+
+      - name: Create Pull Request
+        uses: thomaseizinger/create-pull-request@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          title: "release: ${{ github.event.inputs.versioning }} version up ${{ steps.set-pkg.outputs.name }}"
+          body: "${{ steps.read_file.outputs.content }}"
+          head: ${{ steps.new-branch.outputs.name }}
+          base: main
+          reviewers: "ebma, TorstenStueber, bogdanS98, gianfra-t, b-yap"
```

Comment:
Let's remove the hard-coded reviewers. We can just make it use '@pendulum-chain/devs' manually once the PR is created.

Reply:
updated it.